index.js

```diff
@@ -0,0 +1,62 @@
+
+
+
+function checkForLowerCase(password){
+    for (var i = 0; i < password.length; i++){
+        if (password[i] >= "a" && password[i] <="z"){
+            return true
+        }
+    }return false}
+
+function checkForLength(password){
+    if (password.length < 8) {
+        return false
+    }
+    return true
+}
+
+function checkForUpperCase(password) {
+    for (var i = 0; i < password.length; i++){
```

Comment:
i see you are using for loops inside the methods...

you can instead use for loop at the 'validatePassword' level and pass in each character to the function instead. and if variable already set to true, no need to set them to false again.